src/core/client/stream/classes.ts

```diff
@@ -434,16 +434,22 @@ const CLASSES = {
   },
 
   /**
-   * rejectedTombstone is shown when a comment got rejected.
+   * moderationRejectedTombstone is shown to the moderator when a comment got rejected.
    */
-  rejectedTombstone: {
-    $root: "coral coral-rejectedtombstone",
-    goToModerateButton: "coral coral-rejectedtombstone-goToModerateButton",
+  moderationRejectedTombstone: {
+    $root: "coral coral-moderationRejectedTombstone",
+    goToModerateButton:
+      "coral coral-moderationRejectedTombstone-goToModerateButton",
```

Comment:
just a note, that if we change this, we need to make sure our client CSS that might be using it for custom styling is updated as well.